### PR TITLE
Fix nss3 loading on OS X.

### PR DESCRIPTION
**Pull request by vbadev**
It seems that on OS X full path to libraries is needed.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -81,15 +85,19 @@ function declareFunction(name, library, args) {
 }
 
 function loadLibraries() {
+  let dir = "";
+  if (platformIsOSX) {
```
Comment (mozkeeler) [inline lib/main.js]: This should be 'platformIsOSX()', right?

File: lib/main.js [hunk 2]
```diff
@@ -81,15 +85,19 @@ function declareFunction(name, library, args) {
...
+    dir = require('sdk/system').pathFor("GreBinD") + "/";
```
Comment (mozkeeler) [inline lib/main.js]: Let's hoist all 'require(...)' statements to the top of this file:
```
let system = require("sdk/system");
...
dir = system.pathFor(...
```

**Comment by mozkeeler** [comment]
This looks good - feel free to push another commit addressing the review comments to this branch. I'll merge it when you let me know it's ready to go.

**Comment by vbadev** [comment]
A have fixed problems in new commit.

**Pull request merged**